docs/source/zh/attention.md

```diff
@@ -0,0 +1,36 @@
+<!--版权2023年HuggingFace团队保留所有权利。
+
+根据Apache许可证第2.0版（“许可证”）许可；除非符合许可证，否则您不得使用此文件。您可以在以下网址获取许可证的副本：
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+除非适用法律要求或书面同意，否则按“按原样”分发的软件，无论是明示还是暗示的，都没有任何担保或条件。请参阅许可证以了解特定语言下的权限和限制。
+
+⚠️ 请注意，本文件虽然使用Markdown编写，但包含了特定的语法，适用于我们的doc-builder（类似于MDX），可能无法在您的Markdown查看器中正常渲染。
+
+-->
+
+# 注意力机制
+
+在注意力矩阵是正方形的意义上，大多数变压器模型使用完全注意力。当你有很长的文本时，这可能是一个很大的计算瓶颈。Longformer和reformer是试图提高效率的模型，它们使用注意力矩阵的稀疏版本来加速训练。
```

Comment:
```suggestion
大多数 transformer 模型使用完全注意力机制，该机制采用正方形的注意力矩阵。当输入很长的文本时，这将导致巨大的计算瓶颈。Longformer 和 Reformer 是提高注意力机制效率的改进模型，它们使用稀疏化的注意力矩阵来加速训练。
```

Reply:
Thank you for your advice! I'm sorry that I didn't fully understand this concept, so I translated it like this. I'll revise my translation right away.